### PR TITLE
Banner, AppInfoView: Use a single accent color, auto text color

### PR DESCRIPTION
**Pull request by danrabbit**
Supersedes #1722 

In the interest of adopting the single branding color that's now supported upstream in AppStream: https://github.com/ximion/appstream/issues/187

### CODE REVIEW

File: src/Widgets/Banner.vala [hunk 1]
```diff
@@ -102,24 +102,28 @@ public class AppCenter.Widgets.Banner : Gtk.Button {
...
+                } else {
+                    var bg_rgba = Gdk.RGBA ();
+                    bg_rgba.parse (bg_color);
 
-        try {
+                    text_color = Granite.contrasting_foreground_color (bg_rgba).to_string ();
+                }
+            }
+
+            var colored_css = BANNER_STYLE_CSS.printf (bg_color, text_color);
```
Comment (jeremypw) [inline src/Widgets/Banner.vala:122]: Is it OK to (potentially) print `null` into the css here?

**Review by jeremypw — COMMENTED** [review]
Works as expected during testing.  Just a question about the effect of printfing a null string into the css string.

File: src/Widgets/Banner.vala (hunk 1 above)
Comment (danrabbit) [reply src/Widgets/Banner.vala:121]: Oops you're right, that is problematic. Should be fixed now

**Review by jeremypw — APPROVED** [review]
Good to go now.

**Comment by danrabbit** [comment]
Woo, thanks for thorough review!

**Pull request merged**